docsbox/docs/utils.py

```diff
@@ -238,6 +268,13 @@ def check_file_content(original, converted):
 
     return has_content and original_page_num != 0 and converted_page_num != 0 and original_page_num == converted_page_num
 
+def heic_to_png(path):
+    tmp_file = app.config["MEDIA_PATH"] + "tmp"
+    register_heif_opener()
```

Comment:
This sounds like something we would want to do once during startup, not every time a heif is converted.